newrelic/core/config.py

```diff
@@ -717,7 +717,7 @@ def default_otlp_host(host):
 _settings.transaction_events.attributes.include = []
 
 _settings.custom_insights_events.enabled = True
-_settings.ml_insights_events.enabled = True
+_settings.ml_insights_events.enabled = False
```

Comment:
Is this supposed to stay false too? Or just the global override setting?

Reply:
This one was supposed to stay False since we check this as a standalone setting in calls to `record_ml_event` and not in conjunction with machine_learning.enabled.